e2e/e2eutil/job.go

```diff
@@ -240,6 +240,20 @@ func MaybeCleanupJobsAndGC(jobIDs *[]string) func() {
 	}
 }
 
+// MaybeCleanupNamespacedJobsAndGC stops and purges the list of jobIDs in the namespace and runs a
+// system gc. Returns a func so that the return value can be used
+// in t.Cleanup. Similar to CleanupJobsAndGC, but this one does not assert
+// on a successful stop and gc, which is useful for tests that want to stop and
+// gc the jobs themselves but we want a backup Cleanup just in case.
+func MaybeCleanupNamespacedJobsAndGC(ns string, jobIDs []string) func() {
+	return func() {
+		for _, jobID := range jobIDs {
+			_ = StopJob(jobID, "-namespace", ns, "-purge", "-detach")
```

Comment:
nit: I like test helpers to take a `testing.TB` (interface) and `t.Error` so that failed cleanup doesn't happen invisibly, potentially causing unexpected side effects.